City-of-Helsinki/drupal-hdbt
# UHF-8094: Page template lang support

# [UHF-8094](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8094)
Ensure RTL language support for page templates.

## What was done
* Added some dynamic CSS properties in favor of absolute ones

## How to install

* Set up an instance of your choice, but **preferably** etusivu since it already has support for multiple languages
* Make sure you have arabic or some other RTL language installed
* Install this branch: `composer require drupal/hdbt:dev-UHF-8094-page-template-lang-support`
* Run `drush cr`

## How to test

* Translate a landing page, a standard page and a news item into arabic or some other RTL language
* Check that these template features work in the RTL layout:
  * Table of contents
  * Views on the new item page
* Open the page in incognito mode and check that the cookie banner buttons are properly spaced

Click around the pages and see if anything looks out of place or broken. Is there anything else that we should take into account?



[UHF-8094]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8094?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
